askama_shared/src/filters/mod.rs

```diff
@@ -158,6 +161,14 @@ pub fn indent(s: &fmt::Display, width: &usize) -> Result<String> {
     Ok(indented)
 }
 
+/// Casts number to i32
+pub fn int<T>(number: T) -> Result<i32>
```

Comment:
So this is called `int`, but it casts to `i32` -- which is kind of a random match. It could just as well be `isize`, `usize`, or `u64`.

Reply:
We adjust it to isize.